tests/test_api.rs

```diff
@@ -321,6 +376,7 @@ fn validate_error(instance: &Value, message: Option<&str>) {
             .expect("Error detail")
             .to_string();
 
+        print!("{}", err);
```

Comment:
Is this left over from some debugging?

Reply:
Thanks, changed to a custom panic message instead.